inception-project/inception
# #792 - Open imported doc in external search

Pull request for feature request #792 A quick way to go to document annotation after importing from external search.

**What's in the PR**
* Added an `Open` button in the `SearchPage` `ResultRowView` that becomes visible when a document has been imported
* The `Open` button opens the document in `AnnotationPage`
* Closes #792

**How to test manually**
*Part 1.*
1. Select a project and go to the external search page.
2. Type in a search keyword and wait for the results.
3. From the results, import a document.
4. The `Import` button should disappear and the `Open` button should appear.
5. Click on the `Open` button, the page should redirect to the Annotation page with the imported document open. Check that the title matches the one imported.

*Part 2.*
1. Go back to external search page.
2. Type in the same search keyword as Part 1 and wait for results.
3. Check that the imported document from Part 1 has the `Open` button visible.
4. Click on the `Open` button to check that it opens the document.

**Automatic testing**
None
